lib/puppet_forge/lazy_relations.rb

```diff
@@ -11,9 +11,25 @@ module LazyRelations
     # @private
     def self.included(base)
       base.extend(self)
-      base.after_initialize { @_lazy = {} }
     end
 
+    def parent
+      if self.is_a? Class
+        class_name = self.name
+      else
+        class_name = self.class.name
+      end
+
```

Comment:
So the goal of that function is to go from PuppetForge::V3::Module/User/Release to PuppetForge::V3. I didn't think that was what superclass would do. Is there an easier way to get the module that wraps a class like that?